src/account/account.cairo

```diff
@@ -221,7 +221,7 @@ mod Account {
         assert(self == caller, 'Account: unauthorized');
     }
 
-    #[internal]
+    #[private]
```

Comment:
we should properly define internal and private, and document them in our [styleguide](https://github.com/OpenZeppelin/cairo-contracts/issues/696)

Reply:
https://github.com/OpenZeppelin/cairo-contracts/issues/696#issuecomment-1719193131